app/validators/csv_validator.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class CsvValidator < ActiveModel::Validator
+  CSV_EXTENSION = '.csv'
+
+  def validate(csv_form)
+    validate_file(csv_form)
```

Comment:
```suggestion
    file = csv_form.file
    add_error(csv_form, :wrong_count) unless valid_count(file)
    add_error(csv_form, :wrong_type) unless valid_extension(file)
```

It seems like we do not need an additional method `validate_file` ;-) 
(if needed later we can bring it back)